### DISCUSSION TITLE
Can the installation location be determined dynamically?

### DISCUSSION THREAD

**Discussion by black7375** [wixtoolset/issues#6561] [Questions] [answered]
I'm trying to port a powershell script to a GUI installer.

At this point, I want to ask if it is possible because there is a necessary step.
After parsing the `ini` file, the installation location should be determined by the user's choice. 
  - https://github.com/black7375/Firefox-UI-Fix/blob/7b39b306fd17ef6261198908db4069f0fc2297e4/install.ps1#L419-L473
  
Is it possible?

**Comment by robmen** [comment, marked as answer]
Yes. Depending on where the INI file is stored, you can use IniSearch otherwise you might need a custom action.

**Reply by black7375** [reply]
Thank you. I'll try it soon.
